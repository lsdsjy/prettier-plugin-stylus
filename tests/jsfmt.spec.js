run_spec(__dirname, { tabWidth: 4 });
