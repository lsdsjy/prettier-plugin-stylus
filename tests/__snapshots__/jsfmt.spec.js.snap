// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`blank-lines.styl 1`] = `
.checkbox
    position relative

    &__small
        min-height 20px
    
    &__big
        max-height 100px~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.checkbox
    position relative

    &__small
        min-height 20px

    &__big
        max-height 100px

`;

exports[`import.styl 1`] = `
@import './reset.styl'

@require 'bar.styl'

.foo
  @import "bar.styl"

@media screen and (min-width: 640px)
  @import 'bar.styl'

@import 'product/*'
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
@import './reset.styl'

@require 'bar.styl'

.foo
    @import 'bar.styl'

@media screen and (min-width: 640px)
    @import 'bar.styl'

@import 'product/*'

`;

exports[`keyframes.styl 1`] = `

@keyframes example
    0%
        background #ffffff
    50%
        background #f0f0f0
    to
        color red

$keyframe-name = pulse
@keyframes {$keyframe-name}-a
  for i in 0..10
    {10% * i}
      opacity (i/10)~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

@keyframes example
    0%
        background #ffffff
    50%
        background #f0f0f0
    to
        color red

$keyframe-name = pulse
@keyframes {$keyframe-name}-a
    for i in 0 .. 10
        {10% * i}
            opacity i / 10

`;

exports[`media.styl 1`] = `

@media (-webkit-min-device-pixel-ratio: 2)
    div
        width 200%
        height 200%
        transform scale(.5)


@media screen and (min-width: 600px)
    .foo
      padding 20px

@media (min-width: 100px), (min-height: 200px)
    .foo
        color: #100

@media only screen and (min-width: 320px) and (max-width: 480px) and (resolution: 150dpi) {
    body { line-height: 1.4; }
}

@media not all and (orientation: landscape) {
    body {
        color: blue;
    }
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

@media (-webkit-min-device-pixel-ratio: 2)
    div
        width 200%
        height 200%
        transform scale(0.5)


@media screen and (min-width: 600px)
    .foo
        padding 20px

@media (min-width: 100px), (min-height: 200px)
    .foo
        color #100

@media only screen and (min-width: 320px) and (max-width: 480px) and (resolution: 150dpi)
    body
        line-height 1.4

@media not all and (orientation: landscape)
    body
        color blue

`;

exports[`url.styl 1`] = `
.foo
    background url(img/bar.png) no-repeat

.foo
    background url('img/bar.png') no-repeat~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
.foo
    background url(img/bar.png) no-repeat

.foo
    background url('img/bar.png') no-repeat

`;

exports[`variables.styl 1`] = `

 font-size = 14px

 body
   font font-size Arial, sans-serif
   text-color = white
   p
    color text-color

 #logo
   position: absolute
   top: 50%
   left: 50%
   width: w = 150px
   height: h = 80px
   margin-left: -(w / 2)
   margin-top: -(h / 2)~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

font-size = 14px

body
    font font-size Arial sans-serif
    text-color = white
    p
        color text-color

#logo
    position absolute
    top 50%
    left 50%
    width: w = 150px
    height: h = 80px
    margin-left -(w / 2)
    margin-top -(h / 2)

`;
